### PR TITLE
First PR,Doing some changes with bayonet bloat

### PR DESCRIPTION
**Pull request by Saran6969**
## About The Pull Request
Removes Bayonets and adds knives to certain roles
## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
NCR Trooper changes :Removed Trench tool from both shotgun and reserve loadout,moving it to back pack so every trooper gets it no matter loadout choice.
NCR Corporal changes : Removed Trench tool from the CQC loadout,moving it to back pack(Replacing the bayonet) so every corporal gets it no matter loadout choice.
NCR HT Changes: Removed bayonet from R84 loadout,replacing it with a bowie knife.
NCR LT Changes : Replaced the bayonet knife with a Bowie knife in back pack
NCR Captain Changes : Removed trench knife from the El Capitan Loadout and moved it to back pack so any choice still gives a trench knife.
NCR Combat Medic,Combat engineer,Military police,MO and Sargeant are untouched
NCR Rangers and NCR veteran ranger are also untouched
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -272,7 +272,7 @@ Access
 	suit_store = /obj/item/gun/ballistic/automatic/assault_rifle/infiltrator
 
 	backpack_contents = list(
-		/obj/item/melee/onehanded/knife/bayonet = 1,
+		/obj/item/melee/onehanded/knife/bowie = 1
```
Comment (RonBag) [inline code/modules/jobs/job_types/ncr.dm:275]:
```suggestion
		/obj/item/melee/onehanded/knife/bowie = 1,
```

File: code/modules/jobs/job_types/ncr.dm [hunk 2]
```diff
@@ -982,7 +982,7 @@ Access
 	head = /obj/item/clothing/head/f13/ncr/storm
 	suit = /obj/item/clothing/suit/armor/medium/vest/ncr/mant
 	backpack_contents = list(
-		/obj/item/melee/onehanded/knife/bayonet = 1,
+		/obj/item/shovel/trench = 1
```
Comment (RonBag) [inline code/modules/jobs/job_types/ncr.dm:985]:
```suggestion
		/obj/item/shovel/trench = 1,
```

**Review by RonBag — CHANGES_REQUESTED** [review]

**Comment by Saran6969** [comment]
These are my bad,I should have double checked before I did the pull request

File: code/modules/jobs/job_types/ncr.dm (hunk 2 above)
Comment (Saran6969) [reply code/modules/jobs/job_types/ncr.dm:985]: These are my bad,I should have double checked before I did the pull request

**Review by RonBag — APPROVED** [review]

**Pull request merged**